packages/frontend/package.json

```diff
@@ -16,6 +16,10 @@
     "web-vitals": "^2.1.4"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
```

Comment:
Hey I see some duplicates in frontend and backend. Could you reinstall in root these dependencies?